src/components/SecondaryPanes/index.js

```diff
@@ -198,7 +201,9 @@ class SecondaryPanes extends Component<Props> {
       },
       {
         header: L10N.getStr("callStack.header"),
-        className: "call-stack-pane",
+        className: classnames("call-stack-pane", {
+          inactive: !this.props.pauseData
```

Comment:
We've done this in the past and found that it is better to use react not to render items